spec/requests/transformation_mappings_spec.rb

```diff
@@ -248,6 +275,36 @@ def href_slug(obj)
 
           expect(response).to have_http_status(:ok)
         end
+
+        it "can update a transformation mapping" do
+          api_basic_authorize(action_identifier(:transformation_mappings, :edit, :resource_actions))
+
+          request = {
+            'action'                       => 'edit',
```

Comment:
maybe use " for consistency with the rest of the hash.

Reply:
Sure